package.json

```diff
@@ -10,6 +10,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "lru-cache": "^5.1.1",
```

Comment:
I think it's worth adding a heavily used library here instead of rolling our own.